packages/flutter/lib/src/material/grid_tile_bar.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'icon_theme.dart';
+import 'icon_theme_data.dart';
+import 'typography.dart';
+
+
+/// Typically used to stack a one or two line header or footer on a Grid tile.
+/// The layout is based on the "Grid Lists" section of the Material Design spec:
+/// https://www.google.com/design/spec/components/grid-lists.html#grid-lists-specs
+/// For a one-line header specify title and to add a second line specify caption.
+/// Use left or right to add an icon.
+class GridTileBar extends StatelessComponent {
```

Comment:
Both places where you use this, you have to have a bunch of boilerplate Stack stuff. Would it make sense to have the GridTileBar widget instead take the child to put in the cell as well, and have it be the whole cell instead of just the bottom part?